src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -60,13 +60,13 @@ def create_rename_keys(config, base_model=False):
     )
 
     if base_model:
-        # layernorm + pooler
+        # layernorm
         rename_keys.extend(
             [
                 ("norm.weight", "layernorm.weight"),
                 ("norm.bias", "layernorm.bias"),
-                ("pre_logits.fc.weight", "pooler.dense.weight"),
-                ("pre_logits.fc.bias", "pooler.dense.bias"),
+                # ("pre_logits.fc.weight", "pooler.dense.weight"),
+                # ("pre_logits.fc.bias", "pooler.dense.bias"),
```

Comment:
to remove?

Reply:
I removed support for pre-logits from timm vits because they weren't being used, and was determined in ('How to Train your ViT' and later that they didn't help much). They were only present in some in21k pretrained (original) vits that had zero'ed out classifier heads..